src/reason-parser/reason_ast.ml

```diff
@@ -0,0 +1,110 @@
+open Ast_404
+open Parsetree
+open Asttypes
+open Ast_helper
+
+let parseFastPipe e1 e2 =
+  let attr = (Location.mknoloc "reason.fast_pipe", PStr []) in
```

Comment:
This annotation is probably not necessary right?

Reply:
It is for pretty printing purposes I'm afraid.
`a->f->g` is parsed as an ordinary `g(f(a))`. We need the attribute to give refmt a hint that it actually wasn't `g(f(a))`, but `a->f->g`.